test/hsa/error.jl

```diff
@@ -5,5 +5,5 @@ end
 
 @testset "HSA Async Queue Error" begin
     kernel() = (Device.trap(); nothing)
-    @test_throws Runtime.QueueError wait(@roc kernel())
+    @test_throws Runtime.SignalTimeoutException wait(@roc kernel())
```

Comment:
A trap should trigger the async queue handler, which will then throw a `QueueError` at all running kernels (https://github.com/JuliaGPU/AMDGPU.jl/blob/c85bc0a2ef0b9a939016a71e201c0ded0d1f58c1/src/queue.jl#L140-L146). If this isn't happening, then it's a bug.

Reply:
OK, then it is a bug. Reverted this test.